locomotion-sdk/src/pages/AuthScreens/Onboarding/Email.js

```diff
@@ -37,10 +39,10 @@ const Email = () => {
     try {
       await emailSchema.validateAt('email', { email: value });
     } catch (e) {
-      updateState('email', '');
+      updateState({ email: '' });
       return;
     }
-    updateState('email', email);
+    updateState({ email });
```

Comment:
What is this

Reply:
updates user state, i just changed the way the function takes arguments

Reply:
why not 
```suggestion
setUser(old => {...old, email })
```